Multisig/Data/Services/Utils/DelegateWarningCalculator.swift

```diff
@@ -10,15 +10,17 @@ class DelegateWarningCalculator {
     private static let TRUSTED = false
 
     // MultiSendTx is untrusted: operation = delegate && 'to' address is not 'known'
-        // 'resolving the name': look up in the address info index
+    // 'resolving the name': look up in the address info index
     // Safe Tx is untrusted: operation = delegate && 'to' is not 'known'
-        // 'resolving name': 'to' object's .name is nil
+    // 'resolving name': 'to' object's .name is nil
```

Comment:
the comment should change, too, now it is not synced with the code.

MultiSendTx is untrusted iff there's any "delegate" operation in its transactions.